Analytics/App_Plugins/Analytics/backOffice/AnalyticsTree/directives/localizeOptions.Directive.js

```diff
@@ -0,0 +1,12 @@
+﻿angular.module("umbraco.directives").directive('localizeOption', function (localizationService) {
+    return {
+        restrict: 'A',
+        link: function ($scope, element, attrs, controller) {
+            var key = attrs.localizeOption;
+
+            localizationService.localize(key).then(function (value) {
+                element.text(value);
+            });
+        }
+    }
+});
```

Comment:
This directive handle translations of option values in the dropdownlists. It doesn't like a element inside element: https://our.umbraco.org/forum/umbraco-7/developing-umbraco-7-packages/60131-Localize-in-option-tag-doesnt-work